components/nimbus/src/enrollment.rs

```diff
@@ -897,6 +1087,120 @@ mod tests {
         ]
     }
 
+    fn get_feature_conflict_test_experiments() -> Vec<Experiment> {
+        vec![
+            serde_json::from_value(json!({
+                "schemaVersion": "1.0.0",
+                "slug": "secure-gold",
+                "endDate": null,
+                "featureIds": ["about_welcome"],
+                "branches":[
+                    {
+                        "slug": "control",
+                        "ratio": 1,
+                        "feature": {
+                            "featureId": "about_welcome",
+                            "enabled": false
+                        }
+                    },
+                    {
+                        "slug": "treatment",
+                        "ratio":1,
+                        "feature": {
+                            "featureId": "about_welcome",
+                            "enabled": true
+                        }
+                    }
+                ],
+                "channel": "nightly",
+                "probeSets":[],
+                "startDate":null,
+                "appName": "fenix",
+                "appId": "org.mozilla.fenix",
+                "bucketConfig":{
+                    // Setup to enroll everyone by default.
+                    "count":10_000,
+                    "start":0,
+                    "total":10_000,
+                    "namespace":"secure-gold",
+                    "randomizationUnit":"nimbus_id"
+                },
+                "userFacingName":"Diagnostic test experiment",
+                "referenceBranch":"control",
+                "isEnrollmentPaused":false,
+                "proposedEnrollment":7,
+                "userFacingDescription":"This is a test experiment for diagnostic purposes.",
+                "id":"secure-gold",
+                "last_modified":1_602_197_324_372i64
+            }))
+            .unwrap(),
+            serde_json::from_value(json!({
+                "schemaVersion": "1.0.0",
+                "slug": "secure-silver",
+                "endDate": null,
+                "branches":[
+                    {"slug": "control", "ratio": 1}, // XXX add feature
+                    {"slug": "treatment","ratio":1}, // XXX add feature
```

Comment:
Did you mean to go back and add `about_welcome` here or is this intentionally left out to test something?

Reply:
Fixed.